vmm/src/config.rs

```diff
@@ -864,6 +864,11 @@ impl Default for FsConfig {
 }
 
 impl FsConfig {
+    pub const SYNTAX: &'static str = "FS parameters \
```

Comment:
Could you please copy/paste from what is already in `src/main.rs`?
And don't forget to reuse this new `SYNTAX` from `src/main.rs` too.